vertx-sql-client-template/src/main/java/io/vertx/sqlclient/template/SqlTemplate.java

```diff
@@ -0,0 +1,145 @@
+package io.vertx.sqlclient.template;
```

Comment:
BTW you should have 2020 license headers in files (IntelliJ copyright update can help)